app/components/layer-wrapper/component.js

```diff
@@ -16,6 +16,18 @@ export default Ember.Component.extend({
         let height = this.get('layer.settings.values.height') ? 'height: ' + this.get('layer.settings.values.height') + 'px;' : '';
         return Ember.String.htmlSafe(bgImage + bgCover + height + bgColor + color);
     }),
+    didRender(){
+        let self = this;
+        $.each(this.get('layers'), function( index, layer ) {
+            if(layer.component === 'pages-menu'){
+                $('.pages-menu' ).find('.layer-settings').css({
```

Comment:
when you go in to edit mode the settings appears above the navigation, pushing it below the layer view, this is here so you can see the changes that you are making on the nav